src/com/rakovets/course/challenge/generics/Pair.java

```diff
@@ -0,0 +1,34 @@
+package com.rakovets.course.challenge.generics;
+
+/**
+ * Создать класс Pair, параметризованный двумя параметрами K и V:
+ * хранить две переменные типов K и V соответственно
+ * иметь конструктор, принимающий 2 параметра типов K и V
+ * соответствующие геттеры/сеттеры
+ */
+
+public class Pair<K, V> {
+	K black;
+	V white;
```

Comment:
Bad variables naming
```suggestion
	K key;
	V value;
```